taskfiles/boost/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+cmake_minimum_required(VERSION 3.22.1)
+
+project(boost-test LANGUAGES CXX VERSION 0.1.0)
+
+set(CMAKE_EXPORT_COMPILE_COMMANDS
+    ON
+    CACHE BOOL
+    "Enable/Disable output of compile commands during generation."
+    FORCE
+)
+
+include("build/deps/cmake-settings/all.cmake")
```

Comment:
Set this to be `OPTIONAL` and include the project guard + comment for readability/explanation.
See: https://github.com/y-scope/clp/blob/main/components/core/CMakeLists.txt#L99